parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -13,6 +13,8 @@ RUNC_DOWNLOADS_DIR="/opt/runc/downloads"
 K8S_DOWNLOADS_DIR="/opt/kubernetes/downloads"
 K8S_PRIVATE_PACKAGES_CACHE_DIR="/opt/kubernetes/downloads/private-packages"
 UBUNTU_RELEASE=$(lsb_release -r -s)
+# For Mariner 2.0, this returns "MARINER" and for AzureLinux 3.0, this returns "AZURELINUX"
+OS=$(sort -r /etc/*-release | gawk 'match($0, /^(ID_LIKE=(coreos)|ID=(.*))$/, a) { print toupper(a[2] a[3]); exit }')
```

Comment:
what may happen if it is not mariner or azure linux 3.0, e.g. ubuntu.
Do `/etc/*-release` and `gawk` always exist in our vhd?

Reply:
Hi @fseldow, yeah /etc*-release and gawk exists.

Here is a log [link](https://msazure.visualstudio.com/CloudNativeCompute/_build/results?buildId=102291076&view=logs&j=551b3a3e-dd8c-5d5a-8c9c-6a91c285eec5&t=7ad90e9f-6599-5ef0-a0ea-21905c62d2d0&l=579). In addition, here is a [link ](https://msazure.visualstudio.com/CloudNativeCompute/_build/results?buildId=102291076&view=results)to a vhd test run that runs all the different targets.

Reply:
this is breaking local dev machine on OSX. when we run `make generate` it fai